.expeditor/scripts/ensure-minimum-viable-hab.ps1

```diff
@@ -1,7 +1,8 @@
 [Version]$hab_version = (hab --version).split(" ")[1].split("/")[0]
```

Comment:
Is there any scenario where this actually gets set anymore? My troubleshooting turned up `nil` here.